salt/client/__init__.py

```diff
@@ -567,10 +567,14 @@ def cmd_batch(
         if 'gather_job_timeout' in kwargs:
             opts['gather_job_timeout'] = kwargs['gather_job_timeout']
 
+        eauth=kwargs.get('eauth', {})
```

Comment:
Er, my comment wasn't very clear. It will be called like this:

```
local = salt.client.LocalClient()
local.cmd_batch('*', 'test.ping', username='saltdev', password='saltdev', eauth='auto')
```